proxy/__init__.py

```diff
@@ -23,7 +23,10 @@
 
 class UpstreamRegistryError(Exception):
     def __init__(self, detail):
-        msg = f"Error requesting upstream registry: {detail}"
+        msg = (
+            "the requested image may not exist in the upstream registry, or you do not"
```

Comment:
`you` is misleading there, the client who will read this error may not know how the Quay organization is configured.

Reply:
updated as discussed on slack.